### DISCUSSION TITLE
Direct messages are not being received

### DISCUSSION THREAD

**Discussion by armandkrynauw** [discordjs/discord.js#8864] [Q&A] [answered]
The following event handler does not fire when a user sends direct messages to the bot. However, with my current setup, it does fire when regular messages are sent in a guild. Are there perhaps any options that are missing? Help would be greatly appreciated.

```js
client.on(Events.MessageCreate, msg => {
    // Do stuff with message here
})
```
 **Relevant Client Options**
```js
const client = new Client({
    intents: [
	GatewayIntentBits.Guilds,
	GatewayIntentBits.GuildMessages,
	GatewayIntentBits.DirectMessages,
    ],
    partials: [
	"CHANNEL",
    ],
});
```

**Further Details**
discord.js@14.6.0
node@19.1.0
OS: macOS Ventura 13.0.1

**Comment by almeidx** [comment, marked as answer]
`"CHANNEL"` is not a valid partial in v14. You have to import `Partials` from discord.js and use `Partials.Channel`

**Reply by armandkrynauw** [reply]
This fixed the issue. Thx a lot🙂
